structuretoolkit/analyse/spatial.py

```diff
@@ -517,8 +539,8 @@ def get_voronoi_vertices(
 
 
 def _get_neighbors(
-    structure,
-    position_interpreter,
+    structure: Atoms,
+    position_interpreter: callable,
```

Comment:
You can [hint the callable signature](https://peps.python.org/pep-0677/) (incl. return) directly, if you know it already here.

Reply:
I like the suggestion, still I think it is beyond the scope of this pull request.